renderer/index.js

```diff
@@ -192,6 +201,40 @@ electron.ipcRenderer.on('addFakeDevice', function (e, device) {
   update()
 })
 
+// Load state.saved from the JSON state file
+function loadState () {
+  cfg.read(function (err, data) {
+    if (err) console.error(err)
+    console.log("loaded state: " + JSON.stringify(data, null, 4) /* pretty print */
+    state.saved = data
+    if (!state.saved.torrents) state.saved.torrents = []
+    state.saved.torrents.forEach(startTorrenting)
+  })
+}
+
+// Write state.saved to the JSON state file
+function saveState () {
+  cfg.write(state.save, function (err) {
```

Comment:
typo from last comment is still here. `state.save` should be `state.saved`, right?